packages/cli/src/commands/build.ts

```diff
@@ -0,0 +1,44 @@
+import * as jsYAML from 'js-yaml';
+import { promises as fs } from 'fs';
+import * as path from 'path';
+import * as ora from 'ora';
+import { localModulePath } from '../utils';
+
+export interface BuildCommandOptions {
+  config: string;
+}
+
+const defaultConfig: BuildCommandOptions = {
```

Comment:
NIT: maybe you could rename to `defaultOptions` because the options contain config.

Reply:
Sure, I'll update them.